packages/itinerary-body/src/TransitLegBody/alerts-body.tsx

```diff
@@ -21,6 +21,7 @@ export default function AlertsBody({
   AlertIcon = S.DefaultAlertBodyIcon,
   timeZone = getUserTimezone()
 }: Props): ReactElement {
+  if (typeof alerts !== "object") return null;
```

Comment:
It looks like `alerts` is an array below? If for some reason we need to check for 'object' anyway, I'd add a comment about what's going on

Reply:
`typeof` on an array is `object`! I found that out when I wrote this

Reply:
Javascript is very cool